challenges/objects-arrays.js

```diff
@@ -70,7 +70,10 @@ Name email@example.com
 
 Log the result of your new array. */
 let contactInfo = [];
-contactInfo = graduates.map((name) => `${name.animal_name} ${name.scientific_name}`)
+for (let i = 0; i < graduates.length; i++){
+contactInfo.push(graduates[i].first_name)
```

Comment:
It works and looks good. To make it look the example ( Name email@example.com) It would look like this: contactInfo.push(`${graduates[i].first_name} ${graduates[i].email}`);